packages/composites/google-weather/src/adapter.ts

```diff
@@ -117,7 +117,7 @@ class QueryBuilder {
     this.dateFrom = new Date(dateFrom)
     this.dateTo = new Date(dateTo)
     this.method = method
-    this.column = column
+    this.column = column.replace(/\W/g, '')
```

Comment:
This is only stripping whitespace right? Would it make sense to do a stronger validation for alphanumeric-only?

Reply:
Ahh my bad, just read up on the spec for the `\W` metacharacter. Yeah that looks good!